egs/wsj/s5/steps/nnet3/components.py

```diff
@@ -147,12 +165,14 @@ def AddConvolutionLayer(config_lines, name, input,
 
     num_x_steps = (1 + (input_x_dim - filt_x_dim) / filt_x_step)
     num_y_steps = (1 + (input_y_dim - filt_y_dim) / filt_y_step)
-    output_dim = num_x_steps * num_y_steps * num_filters;
+    num_z_steps = (1 + (input_z_dim - filt_z_dim) / filt_z_step)
+    output_dim = num_x_steps * num_y_steps * num_z_steps * num_filters;
     return {'descriptor':  '{0}_conv_t'.format(name),
             'dimension': output_dim,
-            '3d-dim': [num_x_steps, num_y_steps, num_filters],
+            '3d-dim': [num_x_steps, num_y_steps, num_z_steps * num_filters],
             'vectorization': 'zyx'}
```

Comment:
make it clear that the output is actually a vectorized 4d-tensor but we ignore this.